macros/edr/system/system_utils/get_elementary_package_version.sql

```diff
@@ -0,0 +1,10 @@
+{# Only works if package is pulled from the registry. #}
```

Comment:
What do you mean?

Reply:
You have to use `package` and not `git` or `local` in `packages.yml`.
I'll update the comment.